audio/pavucontrol/Portfile

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           legacysupport 1.1
```

Comment:
Which versions of macOS require legacysupport? Better to specify it rather than rely on the default of the portgroup.

Reply:
It is not dependent on version, it is dependent on `cxx_stdlib`: https://github.com/macports/macports-ports/blob/6df9dbfc009e57fd751890d29566215aae61d461/_resources/port1.0/group/legacysupport-1.1.tcl#L235

Do we have an option not to use it for anything else?

Reply:
It looks like that post-destroot block that does the wrapping is independent of `legacysupport.newest_darwin_requires_legacy`. So I guess if you only want the wrapping and do not want to link with the legacy support library then you should set `legacysupport.newest_darwin_requires_legacy` to `0`.

Reply:
Let me try that.